packages/apollo-server-express/src/ApolloServer.ts

```diff
@@ -73,6 +73,7 @@ const fileUploadMiddleware = (
 export interface ExpressContext {
   req: express.Request;
   res: express.Response;
+  connection: any;
```

Comment:
I don't think that `connection` is guaranteed to be present on the `ExpressContext`, but rather only present when using subscriptions?

https://github.com/apollographql/apollo-server/blob/489cbb42cd6cd5c50c60cfaca93060912c44d20e/packages/apollo-server-core/src/ApolloServer.ts#L474

That's to say, I think this should be _optional_ (i.e. `connection?`)?

Reply:
Additionally, rather than `any`, would it be appropriate to be `ExecutionParams`?:

https://github.com/apollographql/apollo-server/blob/489cbb42cd6cd5c50c60cfaca93060912c44d20e/packages/apollo-server-core/src/ApolloServer.ts#L463

(And please forgive my unfamiliarity with subscriptions in general, even though I know they are an important part of many user's projects, Apollo Server hasn't quite arrived at a fully-integrated, first-class subscriptions offering, though it's something we intend on improving!)

```suggestion
  connection?: ExecutionParams;
```